tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_bluetooth.py

```diff
@@ -33,7 +32,10 @@ def test_toggle_bluetooth_settings(self):
 
         self.assertEquals(bluetooth_settings.device_name, device_name)
 
-        bluetooth_settings.enable_visible_to_all()
+        bluetooth_settings.enable_visible_to_all(self.data_layer)
 
         self.assertTrue(self.data_layer.bluetooth_is_discoverable)
```

Comment:
should remove this line, (if we are certain that we can't check the state of the checkbox)